data-mermaid/mermaid-webapp
# Bleaching summary bug fixes

Requesting to merge into your branch @mmnoo since it hasn't been merged in yet. 

Added two bug fixes for NaN and for the wonky averages (needed to convert string to Number)

to test:

1. go to bleaching summaries
2. view empty observation tables
3. 0 will display instead of NaN
4. add observation items for percentage cover
5. averages should make sense
